### PR TITLE
add user

### PR DESCRIPTION
**Pull request by hmberger**

### CODE REVIEW

File: data/roster.csv [hunk 1]
```diff
@@ -1,2 +1,2 @@
 github_username,full_name,exercises_website
-bbest,Ben Best,https://bbest.github.io/r3-exercises
+hmberger,Halle Berger,https://hmberger.github.io/r3-exercises
```
Comment (bbest) [inline data/roster.csv:2]: Hi @hmberger, this is the only change I was looking for in this Pull Request

**Review by bbest — CHANGES_REQUESTED** [review]
Hi @hmberger,

It looks like you performed [1.7 Setup Github Pages website](https://noaa-iea.github.io/r3-train/collaborate.html#17_Setup_Github_Pages_website) in the incorrect repo [github.com/hmberger/**r3-train**](https://github.com/hmberger/r3-train) (the forked copy of [github.com/**noaa-iea**/r3-train](https://github.com/noaa-iea/r3-train)) as well the appropriate [github.com/hmberger/r3-exercises](https://github.com/hmberger/r3-exercises).

For this Pull Request, I was only looking for the extra line in `data/roster.csv` to be changed, per 
 [1.8 Contribute website with `fork` & `pull request`](https://noaa-iea.github.io/r3-train/collaborate.html#18_Contribute_website_with_fork__pull_request). The easiest way to fix this Pull Request would be to:

1. In your r3-train repo, copy your data/roster.csv to somewhere outside the repo
1. Perform a hard reset in RStudio Terminal to before you made changes based on the history of [Commits](https://github.com/hmberger/r3-train/commits/master):
    ```bash
    git reset --hard 0d1d7fc32
    ```
1. Copy your data/roster.csv back into the repo 
1. Perform a Git Commit & Push

Since this Pull Request is connected to the repo [github.com/hmberger/r3-train](https://github.com/hmberger/r3-train), when you update it, changes should show up here.

This is a great way to learn ;)

**Comment by hmberger** [comment]
Yes I realized after that I was in the wrong repo, thanks for your help!

**Comment by bbest** [comment]
Ah woops, I see you did it correctly in Pull Request #7

**Pull request closed without merging**